.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .pytest_cache/
 **__pycache__/
 db.sqlite3
+taxi_service_db_data.json
```

Comment:
Ensure that `taxi_service_db_data.json` is intentionally ignored. If this file contains sensitive data or environment-specific configurations, it makes sense to ignore it. Otherwise, consider whether it should be included in version control.